src/Connection.php

```diff
@@ -1030,7 +1030,7 @@ public function lastInsertId()
      *
      * @throws Throwable
      */
-    public function transactional(Closure $func)
+    public function transactional(Closure $func): mixed
```

Comment:
We probably don't want to do it now but a method with a `mixed` return type looks nasty. It should be something like:
```php
public function transactional(callable:T $func): T;
```

Reply:
PHP still does not support generics on the language level, so we would need to do that in the doc block. Probably something we can do on 3.3.x already.

Reply:
Sorry, I meant a doc block but was lazy to type the whole thing out. Now I realize that in the context of switching to native type declarations it looked confusing.